backends-common/cassandra/pom.xml

```diff
@@ -168,13 +168,25 @@
                     <version>${assertj-3.version}</version>
                     <scope>test</scope>
                 </dependency>
-		        <dependency>
-		            <groupId>org.cassandraunit</groupId>
-		            <artifactId>cassandra-unit</artifactId>
-		            <version>2.1.9.2</version>
-		            <scope>test</scope>
-		        </dependency>
-		    </dependencies>
+                <dependency>
+                    <groupId>org.cassandraunit</groupId>
+                    <artifactId>cassandra-unit</artifactId>
+                    <version>2.1.9.2</version>
+                    <scope>test</scope>
+                </dependency>
+                <dependency>
+                    <groupId>org.mockito</groupId>
+                    <artifactId>mockito-core</artifactId>
+                    <version>1.10.19</version>
+                    <scope>test</scope>
+                </dependency>
+                <dependency>
+                    <groupId>uk.org.lidalia</groupId>
+                    <artifactId>slf4j-test</artifactId>
```

Comment:
ok, so you want to test logging ? we never did that in the past, i'm not against that but is logging so important that we want to make it hard to change ? It's clearly a team decision in my opinion.
Also, does importing slf4j-test prevent logback output ?